src/main/java/com/ttrides/turntablRides/controllers/UserController.java

```diff
@@ -1,16 +1,19 @@
 package com.ttrides.turntablRides.controllers;
 
+import com.ttrides.turntablRides.models.response.TTResponse;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 
 @RestController
-@RequestMapping("/api/v1/login")
```

Comment:
You took out the /login part of the endpoint but you still have your test using /login.

Reply:
No please, it wasn't taken out. I just separated the RequestMapping from GetMapping so, we can join other endpoints to the main user request later.

Because we would have smaller endpoints for the user later, I broke that **/login** just for the test [ @GetMapping(value = "/login") ]